modular_pariah/modules/contractor/code/datums/contract.dm

```diff
@@ -70,167 +88,187 @@
+	target.reagents.add_reagent(/datum/reagent/medicine/omnizine, 20)
+	victim_stage_one(target)
+
+/datum/syndicate_contract/proc/victim_stage_one(mob/living/target)
+	target.flash_act()
+	target.add_confusion(10)
+	target.blur_eyes(5)
+	to_chat(target, span_warning("You feel strange..."))
+	addtimer(CALLBACK(src, .proc/victim_stage_two, target), 6 SECONDS)
+
+/datum/syndicate_contract/proc/victim_stage_two(mob/living/target)
+	to_chat(target, span_warning("That pod did something to you..."))
+	target.dizziness += 10
+	addtimer(CALLBACK(src, .proc/victim_stage_three, target), 6 SECONDS)
+
+/datum/syndicate_contract/proc/victim_stage_three(mob/living/target)
+	to_chat(target, span_warning("Your head pounds... It feels like it's going to burst out your skull!"))
+	target.flash_act()
+	target.add_confusion(20)
+	target.blur_eyes(3)
+	addtimer(CALLBACK(src, .proc/victim_stage_four, target), 3 SECONDS)
+
+/datum/syndicate_contract/proc/victim_stage_four(mob/living/target)
+	to_chat(target, span_warning("Your head pounds..."))
+	addtimer(CALLBACK(src, .proc/victim_stage_five, target), 10 SECONDS)
+
+/datum/syndicate_contract/proc/victim_stage_five(mob/living/target)
+	target.flash_act()
+	target.Unconscious(200)
+	to_chat(target, span_hypnophrase(span_reallybig(">A million voices echo in your head... <i>\"Your mind held many valuable secrets - \
+				we thank you for providing them. Your value is expended, and you will be ransomed back to your station. We always get paid, \
+				so it's only a matter of time before we ship you back...\"</i>")))
+	target.blur_eyes(10)
+	target.dizziness += 30
+	target.add_confusion(20)
+
+/// We're returning the victim
+/datum/syndicate_contract/proc/return_victim(mob/living/target)
 	var/list/possible_drop_loc = list()
 
-	for (var/turf/possible_drop in contract.dropoff.contents)
-		if (!isspaceturf(possible_drop) && !isclosedturf(possible_drop))
-			if (!possible_drop.is_blocked_turf())
-				possible_drop_loc.Add(possible_drop)
+	for(var/turf/possible_drop in contract.dropoff.contents)
+		if(!(!isspaceturf(possible_drop) && !isclosedturf(possible_drop)))
+			continue
+		if(!possible_drop.is_blocked_turf())
+			possible_drop_loc.Add(possible_drop)
 
-	if (possible_drop_loc.len > 0)
-		var/pod_rand_loc = rand(1, possible_drop_loc.len)
+	if (length(possible_drop_loc) > 0)
+		var/pod_rand_loc = rand(1, length(possible_drop_loc))
 
 		var/obj/structure/closet/supplypod/return_pod = new()
 		return_pod.bluespace = TRUE
 		return_pod.explosionSize = list(0,0,0,0)
 		return_pod.style = STYLE_SYNDICATE
 
-		do_sparks(8, FALSE, M)
-		M.visible_message(span_notice("[M] vanishes..."))
+		do_sparks(8, FALSE, target)
+		target.visible_message(span_notice("[target] vanishes..."))
 
-		for(var/obj/item/W in M)
-			if (ishuman(M))
-				var/mob/living/carbon/human/H = M
-				if(W == H.w_uniform)
+		for(var/obj/item/target_item in target)
+			if(ishuman(target))
```

Comment:
Why would they ever not be a human

Reply:
You can shove any mob into these, which sends it off but fails the contract